Subscriber/Backend/RemoveStoredPaymentsSubscriber.php

```diff
@@ -36,15 +36,10 @@ private function removeHiddenAdyenPaymentMethods(Enlight_Controller_Action $subj
 
         $adyenSourceType = SourceType::adyen()->getType();
         foreach ($data as $key => $paymentMethod) {
-            if (false === (bool) $paymentMethod['hide']) {
-                continue;
-            }
-
-            if ($adyenSourceType !== (string) $paymentMethod['source']) {
-                continue;
+            if ((true === $paymentMethod['hide']
```

Comment:
Any reason why this is a combined if?

With early returns / continue the functionality is more clear upon reading the code.